cypress/integration/landing-page.js

```diff
@@ -0,0 +1,68 @@
+describe('Sol Aware home page', () => {
+    beforeEach(() => {
+        cy.visit('http://localhost:3000')
+    })
+
+    it('should have a Header component', () => {
+        cy.get('[data-cy=header]')
+            .should('exist')
+    })
+
+    it('should have a logo with site title', () => {
+        cy.get('[data-cy=logo')
+            .should('exist')
+    })
+
+    it('should have 3 navigation links', () => {
```

Comment:
Maybe add a `.length` to test for the three navigation links?

Reply:
Nevermind, I see that you have this in your next test - maybe just update the it statement to not include a test for three nav links since that is within your next test as opposed to this one.

Reply:
Good call, thank you!